docs/DeveloperGuide.md

```diff
@@ -463,16 +461,69 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
     Use case ends.
     
-**Use case : view modules**
+**Use case : UC04 View all current modules**
 
 **MSS**
 
-1. User requests to view current list of Modules
-2. GradPad shows current list of Modules
+1. User requests to view list of modules in `Current Modules`
+2. GradPad shows all modules added into `Current Modules`
+
+    Use case ends.
+    
+**User case: UC05 Edit a Module in `Current Modules`**
+
+1. User requests to list all modules in `Current Modules`
+2. GradPad shows the list of modules in `Current Modules`
+3. User requests to edit a module in `Current Modules`
```

Comment:
I think we also need an extension to cover the case when invalid fields are given e.g. `credit/5` instead of `cr/5`